src/plays/expanding-cards/ExpandingCards.css

```diff
@@ -26,8 +26,10 @@
   -webkit-backface-visibility: hidden;
 }
 
-.card-container .active {
-  width: 600px;
+@media only screen and (min-width: 600px) {
```

Comment:
unnecessary media query. There is already a section where all media queries lie at the bottom. you either need to 
```
.card-container .active {
    width:100%;
    max-width:600px
  }
or just add the width:100% in media query of same class that has been declared at the bottom.

Reply:
👍 I will make the adjustment. Thanks for the feedback.